{{cookiecutter.app_name}}/README.md

```diff
@@ -51,7 +51,7 @@ pipenv shell
 pip install -r requirements/dev.txt
 {%- endif %}
 npm install
-npm start  # run the webpack dev server and flask server using concurrently
+npm start  # run the webpack dev server and flask server using concurrently (use `start-windows` in windows)
```

Comment:
Rather that having different commands depending on OS, what do you think about using a package like [`run-script-os`](https://www.npmjs.com/package/run-script-os) that allows for OS-aware scripts?

Reply:
If ok with the dependency; have updated